config/environments/production.rb

```diff
@@ -70,7 +70,16 @@
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = YAML.load(File.read('config/mailer.yml'))[Rails.env].symbolize_keys
+  config.action_mailer.smtp_settings = {
+    enable_starttls_auto: ENV['MAILER_ENABLE_STARTTLS_AUTO'] || true,
+    address: ENV['MAILER_ADDRESS'],
+    port: ENV['MAILER_PORT'].to_i || 587,
```

Comment:
not blocking: normal practice is to use ENV.fetch('KEY', 'default_value')
```suggestion
    port: ENV.fetch('MAILER_PORT', 587).to_i
```